src/plugins/intel_npu/tools/protopipe/src/parser/config.cpp

```diff
@@ -175,9 +193,12 @@ struct convert<LayerVariantAttr<T>> {
     }
 };
 
+static const std::set<std::string> set_UniformGenerator_parameters = {"low", "high"};
 template <>
 struct convert<UniformGenerator::Ptr> {
     static bool decode(const Node& node, UniformGenerator::Ptr& generator) {
+        std::cout << "Septi: " << __FILE__ << ":" << __LINE__ << "   ---   " << __func__ << std::endl;
```

Comment:
Please remove in final code review

Reply:
This line is needed for debug. It will be removed when we reach an final agreement.